src/lib/packet.h

```diff
@@ -565,21 +572,44 @@ typedef enum {
 
 #define PGP_SA_DEFAULT_CIPHER PGP_SA_CAST5
 
+#define PGP_SA_DEFAULT_BLOCK_CIPHER_MODE PGP_BLOCK_CIPHER_MODE_CFB
+
 void pgp_calc_mdc_hash(
   const uint8_t *, const size_t, const uint8_t *, const unsigned, uint8_t *);
 unsigned pgp_is_hash_alg_supported(const pgp_hash_alg_t *);
 
+/* Maximum block size for symmetric crypto */
+#define PGP_MAX_BLOCK_SIZE 16
+
+/* Maximum key size for symmetric crypto */
+#define PGP_MAX_KEY_SIZE 32
+
+/* Salt size for hashing */
+#define PGP_SALT_SIZE 8
```

Comment:
I think you accidentally duplicated a few of theses defines in packet.h. `PGP_MAX_BLOCK_SIZE` `PGP_MAX_KEY_SIZE` `PGP_SALT_SIZE` are duplicated.